compiler/rustc_codegen_ssa/src/codegen_attrs.rs

```diff
@@ -122,6 +122,28 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, did: LocalDefId) -> CodegenFnAttrs {
                 }
                 AttributeKind::Cold(_) => codegen_fn_attrs.flags |= CodegenFnAttrFlags::COLD,
                 AttributeKind::Align { align, .. } => codegen_fn_attrs.alignment = Some(*align),
+                AttributeKind::NoMangle(attr_span) => {
+                    no_mangle_span = Some(*attr_span);
```

Comment:
is this no_mangle span still the nicest way to do things?

Reply:
Fixed, cleaned it up a bit